tests/test_python_pr_unit.py

```
@@ -0,0 +1,433 @@
+def load_imagestream():
+    with open(IMAGESTREAM_PATH) as f:
+        return yaml.safe_load(f)
+
+
+def imagestream_tags_by_name(imagestream):
+    """Return a dict mapping tag name -> tag dict."""
+    return {tag["name"]: tag for tag in imagestream["spec"]["tags"]}
+
+
+# ---------------------------------------------------------------------------
+# values.schema.json – python_version enum
```

Ruff (0.15.15)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)